pages/mod/_id/edit.vue

```diff
@@ -491,9 +502,13 @@ export default {
         }
 
         this.isEditing = false
-        await this.$router.replace(
-          `/mod/${this.mod.slug ? this.mod.slug : this.mod.id}`
-        )
+        this.savedMod = this.mod
+
+        this.$notify({
+          group: 'main',
+          title: 'Changes saved',
+          type: 'success',
+        })
```

Comment:
What's the point of changing this?

Reply:
It makes more sense to me, that the user should stay on the project editing page rather than be redirected after saving. The user only clicked `Save`, not `Save & view result`.